tests/python/qe-core/openbroadcastersoftware.py

```diff
@@ -0,0 +1,35 @@
+from testapi import *
+
+def run(self):
+
+    perl.require("serial_terminal")
+    perl.require("utils")
+    perl.require("x11utils")
+
+
+    perl.serial_terminal.select_serial_terminal()
+    perl.utils.zypper_call("in flatpak")
+    assert_script_run("flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo")
+    assert_script_run("flatpak install -y com.obsproject.Studio",300)
```

Comment:
Any particular reason why using flatpak vs using the obs from repos?

Reply:
I haven't found official repos, see https://software.opensuse.org/package/obs-studio and https://github.com/obsproject/obs-studio/wiki/install-instructions#unofficial-builds